parse/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "phylum-parse"
+description = "Parsers used by Phylum"
+version = "0.1.0"
+authors = ["Phylum, Inc. <engineering@phylum.io>"]
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
This comment is unnecessary.

We should probably also specify MSRV considering we do so in cli too.